Header header
std_msgs/Time scan_end_time
int64 workspace_id
geometry_msgs/PoseStamped workspace_pose
int64[] people_ids
geometry_msgs/PoseStamped[] people_poses
